analytics/views/remote_activity.py

```diff
@@ -78,28 +102,60 @@ def get_remote_server_activity(request: HttpRequest) -> HttpResponse:
     LAST_AUDIT_LOG_DATE = 4
     MOBILE_USER_COUNT = 5
     MOBILE_PUSH_COUNT = 6
+    REALM_ID = 7
+    ORG_TYPE = 9
+    ARR = 12
+    TOTAL_USER_COUNT = 13
+    GUEST_COUNT = 14
 
     rows = get_query_data(query)
-    total_row = []
     plan_data_by_remote_server = get_plan_data_by_remote_server()
+    plan_data_by_remote_realm = get_plan_data_by_remote_realm()
+
+    total_row = []
+    server_mobile_data_counted = []
+    total_revenue = 0
+    total_mobile_users = 0
+    total_pushes = 0
 
     for row in rows:
-        # Add estimated revenue for server
-        server_plan_data = plan_data_by_remote_server.get(row[SERVER_ID])
-        if server_plan_data is None:
+        # Count mobile users and pushes forwarded, once per server
+        if row[SERVER_ID] not in server_mobile_data_counted:
+            if row[MOBILE_USER_COUNT] is not None:
+                total_mobile_users += row[MOBILE_USER_COUNT]  # nocoverage
+            if row[MOBILE_PUSH_COUNT] is not None:
+                total_pushes += row[MOBILE_PUSH_COUNT]  # nocoverage
+            server_mobile_data_counted.append(row[SERVER_ID])
+        if row[REALM_ID] is None:
+            plan_data = plan_data_by_remote_server.get(row[SERVER_ID])
+            user_counts = get_remote_server_guest_and_non_guest_count(row[SERVER_ID])
+        else:
+            server_remote_realms_data = plan_data_by_remote_realm.get(row[SERVER_ID])
```

Comment:
Should this be called just `remote_realm_plan_data`? I'm not sure I understand the variable name.

Reply:
Renamed as suggested.